File: beginner/javascript/1151.js
var input = require('fs').readFileSync('stdin', 'utf8');
var valores = input.split('\n');

var x, y, a = 0, b = 1, c = 0, str = '';

x = parseInt(valores.shift());

for (y = 0; y < x; y++) {
    if (y === 0) {
        str += `${a} `;
    } else if (y === 1) {
        str += `${b} `;
    } else {
        c = a + b;
        a = b;
        b = c;
        str += `${c} `;
    }
}

console.log(str.trim());

File: beginner/python/1151.py
x = int(input())

a, b, c = 0, 1, 0

for y in range(1, x):
    if y % 2 == 1:
        print(c, end=" ")
        c = a + b
        a = c
    elif y == 2:
        print(c, end=" ")
    elif y % 2 == 0:
        print(c, end=" ")
        c = a + b
        b = c

print(c)